agents/mysql/perfschema/perfschema_test.go

```diff
@@ -381,4 +389,50 @@ func TestPerfSchema(t *testing.T) {
 		expected.Common.Queryid = digests[expected.Common.Fingerprint]
 		tests.AssertBucketsEqual(t, expected, actual)
 	})
+
+	t.Run("Invalid UTF-8", func(t *testing.T) {
+		m := setup(t, db)
+
+		_, err := db.Exec("SELECT /* AllCities */ * FROM city where Name='BÃ¼rki'")
```

Comment:
⚠️ That's valid UTF-8 byte sequence: https://play.golang.org/p/iJsIPPkI2Ov I bet old code would pass that test too.

Reply:
Interesting thing here that channel is failing if we don't pass it through our truncate.Query function.

Reply:
That sounds concerning. Let's _do_ spend time on investigating that :) Maybe create two different tests.

Reply:
checked it again and yeah, you are right, but for the some reason perfschema truncates query if there is some non-utf8 character. Trying to investigate it.

Reply:
So I'm going to remove this tests because it doesn't make sense.

Reply:
I think we can keep this or similar (with \xff) test to track MySQL and MariaDB behaviour. It's just that it is not "Invalid UTF-8".